go.sum

```diff
@@ -502,8 +502,9 @@ github.com/opencontainers/go-digest v1.0.0 h1:apOUWs51W5PlhuyGyz9FCeeBIOUDA/6nW8
 github.com/opencontainers/go-digest v1.0.0/go.mod h1:0JzlMkj0TRzQZfJkVvzbP0HBR3IKzErnv2BNG4W4MAM=
 github.com/opencontainers/image-spec v1.0.2 h1:9yCKha/T5XdGtO0q9Q9a6T5NUCsTn/DrBg0D7ufOcFM=
 github.com/opencontainers/image-spec v1.0.2/go.mod h1:BtxoFyWECRxE4U/7sNtV5W15zMzWCbyJoFRP3s7yZA0=
-github.com/opencontainers/runc v1.1.3 h1:vIXrkId+0/J2Ymu2m7VjGvbSlAId9XNRPhn2p4b+d8w=
 github.com/opencontainers/runc v1.1.3/go.mod h1:1J5XiS+vdZ3wCyZybsuxXZWGrgSr8fFJHLXuG2PsnNg=
```

Comment:
@mpatlasov Hi Seems there left one more runc v1.1.3 info.

Reply:
@chao007 We control only `go.mod`, while `go.sum` is auto-generated by `go mod tidy; go mod vendor`. Go uses an algorithm called Minimal version selection (https://go.dev/ref/mod#minimal-version-selection) which can provide stale results, we have seen similar behavior in other repos too. The presence of "runc v1.1.3" in `go.sum` doesn't necessarily mean that we depends on that version, it is `go.mod` that matters.